### ISSUE TITLE
Failed Accept4 in Python on 1.12 when using Keras in same program

### ISSUE THREAD

**Issue by pascalwhoop** [grpc/grpc#15756]
v.1.12.1 with python on 18.04 ubuntu
gcc 7.3.0
 
 
### What did you do?
I am running GRPC (server) and Keras in a program. The program receives messages via GRPC and then passes them to a Keras model. the Keras Model is using sequence generators and multiprocessing. The server has 1 cllient connected with 3 open response streams that are waiting for a blocking queue to yield more items. 
 
### What did you expect to see?
 no interaction
 
### What did you see instead?
When `with closing(self.executor_fn(_SHARED_SEQUENCES)) as executor:` is called in `data_utils.py` of Keras, I receive 

```
E0614 11:10:36.290798372   15677 tcp_server_posix.cc:215]    Failed accept4: Invalid argument
E0614 11:10:36.290820014   15677 tcp_server_posix.cc:215]    Failed accept4: Invalid argument
```
and the entire GRPC server is crashed. Unfortunately that's all the logs I get. 
 
I'll ping someone from keras, so they can answer some questions that might arise

**Comment by srini100** [comment]
@dgquintas, can you please check if anything obvious here?

**Comment by pascalwhoop** [comment]
Before trying to build a minimal prototype that reproduces the issue which I can put on pastebin etc, I want to hear if this is generally a possibility as both tools use protobufs and it could be a dependency issue that one can run into without intending to. Or it's something on the network stack, although I wouldn't know why keras/tensorflow uses the network stack. 

It's [this project] (https://github.com/pascalwhoop/broker-python/blob/master/agent_components/demand/estimator.py#L73) where it occurs

**Comment by dgquintas** [comment]
I remember [a similar issue](https://github.com/grpc/grpc/issues/10002) from about a year ago @yang-g debugged. @yang-g , was it fixed somehow or did we just stop seeing the error?

**Comment by yang-g** [comment]
The previous issue was a crash that was fixed. This could be something similar and some repro or debug would be helpful.

**Comment by ericgribkoff** [comment]
This is very likely the same issue reported in https://github.com/grpc/grpc/issues/17093

**Comment by ericgribkoff** [comment]
The fix for this (#17444) is now merged and will be available in the next release.
